README.md

```diff
@@ -187,19 +187,22 @@ import com.opentok.Archive;
 import com.opentok.ArchiveProperties;
 
 Archive archive = opentok.startArchive(sessionId, new ArchiveProperties.Builder()
-  .archiveMode(Archive.OutputMode.INDIVIDUAL)
+  .outputMode(Archive.OutputMode.INDIVIDUAL)
   .build()););
```

Comment:
There is extra `);` needs to be removed from the startArchive.

Reply:
Good catch @Anshul14Sharma  - We just need to remove the extra `)` after the `build`
```
                    archive = opentok.startArchive(sessionId, new ArchiveProperties.Builder()
                                            .outputMode(outputMode)
                                            .build(););
```